lib/resources/datav2/rest_v2.js

```diff
@@ -65,6 +64,30 @@ async function* getDataV2(endpoint, symbol, options = {}, config) {
   }
 }
 
+async function* getMultiDataV2(endpoint, options = {}, config) {
+  let pageToken = null;
+  while (true) {
+    let params = options;
+    Object.assign(params, { limit: options.limit, page_token: pageToken });
```

Comment:
Isn't this limit is inconsistent with the single query limit? I'm pretty sure you're setting the page limit here, and the total limit there.

Reply:
also modern syntax here would be
```js
const params = {...options, limit: options.limit, page_token: pageToken }
```